core/jazz_t-vault/config/dev-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "T_VAULT_API": "{tvault-host-name}/vault/v2",
+  "T_VAULT_USERNAME": "{tvault-username}",
+  "T_VAULT_PASSWORD": "{tvault-password}",
+  "T_VAULT_LOGIN_API": "/auth/tvault/login",
+  "SAFE_API": "/ss/sdb",
+  "USER_IN_SAFE_API": "/user",
+  "ROLE_IN_SAFE_API": "/role",
+  "DELETE_SAFE": "/delete?path=shared/",
+  "GET_SAFE_DETAILS": "?path=shared/",
+  "GET_ROLE_DETAILS": "/auth/aws/role/",
+  "CREATE_ROLE_IN_VAULT": "/ss/auth/aws/iam/role?path=shared/",
+  "USER_IN_VAULT": "/auth/userpass/users"
```

Comment:
Move these under a key (`API`) to keep them separate from other configurations.

```
{
    "API": {
        "SAFE": "/ss/sdb",
        "SAFE_USERS": "/user",
        "SAFE_ROLES": "/role",
        "DELETE_SAFE": "/delete?path=shared/",
        "GET_SAFE": "?path=shared/",
        "GET_ROLE": "/auth/aws/role/",
        "CREATE_ROLE": "/ss/auth/aws/iam/role?path=shared/",
        "USERS": "/auth/userpass/users"
    }
}
```

Reply:
Done